docs/pages/fragments/install/cli.mdx

```diff
@@ -25,6 +25,10 @@ curl -L -o vcluster "https://github.com/loft-sh/vcluster/releases/latest/downloa
 ```bash
 curl -L -o vcluster "https://github.com/loft-sh/vcluster/releases/latest/download/vcluster-darwin-arm64" && sudo install -c -m 0755 vcluster /usr/local/bin && rm -f vcluster
 ```
+Or install using Homebrew:
+```bash
+brew install vcluster
+```
```

Comment:
I was hesitant in making the change for Intel Macs without actually testing that the Homebrew formula actually works on those devices.

Re: Linux, looks like [Linux Brew has been merged into Homebrew](https://github.com/Linuxbrew/brew#linuxbrew-has-been-merged-into-homebrew), so Homebrew should be fine as the label. But again, I will be hesitant in adding it as I can't test it. 

However, I can add the Homebrew instructions for both Intel Macs and Linux without testing if you give a go-ahead.

Reply:
looking at the formula here: https://github.com/Homebrew/homebrew-core/pull/99552/files

It seems to be building from source, so should work for Intel macs too. What I would suggest is instead of editing each section with "OR" we can have a completely new tab saying "Brew" and let the decision be on the user to either use brew or not. I can test on brew with linux in a bit using a vm just to be sure

Reply:
@ishankhare07 - Any further thoughts?

Reply:
hey @architchandra , I think the best is to have a separate tab for brew installation without specifically adding sections for each platform. The formula should take care of building it for the respective ones. So I would suggest you to change this and have just 1 tab for "installation with brew"

Reply:
@architchandra are you still taking a look at this?

Reply:
@ishankhare07 - Hey, I was away for a few days. I'll make the change today.

PS: I still disagree with the design decision, even if some places in the Kubernetes docs have also used this pattern. Will go ahead and make the change, though. :)

Reply:
@ishankhare07 - I've moved the Homebrew instructions to a new tabs but the design looks a bit skewed now with a fifth tab in the section. Please let me know what you think.
![Screenshot 2023-07-17 at 6 23 58 PM](https://github.com/loft-sh/vcluster/assets/8830075/f8c50037-976d-40d6-a400-361939ad3f31)

Reply:
Hey @architchandra , I don't think this is a big problem. We can either:
1. Move the homebrew tab to the last rightmost position
2. We can change the tab label to read "Homebrew installation" so that it looks less skewed

Reply:
@ishankhare07 - I've gone for option 2. Please see if things are good enough to merge now.